File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/delete/DelStatus.java
package ru.citeck.ecos.records3.record.op.delete;

public enum DelStatus {
    /**
     * Deletion was successful
     */
    OK,
    /**
     * Internal error
     */
    ERROR,
    /**
     * Record is protected from deletion
     */
    PROTECTED,
    /**
     * Operation is not supported
     */
    NOT_SUPPORTED
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/delete/dto/DelStatus.kt
package ru.citeck.ecos.records3.record.op.delete.dto

enum class DelStatus {
    /**
     * Deletion was successful
     */
    OK,

    /**
     * Internal error
     */
    ERROR,

    /**
     * Record is protected from deletion
     */
    PROTECTED,

    /**
     * Operation is not supported
     */
    NOT_SUPPORTED
}
